server/graphql/v2/query/ExpenseQuery.js

```diff
@@ -27,7 +27,8 @@ const ExpenseQuery = {
       if (
         expense?.status === expenseStatus.DRAFT &&
         expense.data?.draftKey !== args.draftKey &&
-        !req.remoteUser?.isAdmin(expense.FromCollectiveId)
+        !req.remoteUser?.isAdmin(expense.FromCollectiveId) &&
+        !req.remoteUser?.isAdmin(expense.CollectiveId)
```

Comment:
Should be `!req.remoteUser?.isAdminOfCollective(expense.collective)` to make sure we cover events too